webapp/channels/src/components/signup/signup.tsx

```diff
@@ -136,12 +140,24 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
     const [teamName, setTeamName] = useState(parsedTeamName ?? '');
     const [alertBanner, setAlertBanner] = useState<AlertBannerProps | null>(null);
     const [isMobileView, setIsMobileView] = useState(false);
+    const [subscribeToSecurityNewsletter, setSubscribeToSecurityNewsletter] = useState(false);
+
+    const canReachCWS = useCWSAvailabilityCheck();
 
     const enableExternalSignup = enableSignUpWithGitLab || enableSignUpWithOffice365 || enableSignUpWithGoogle || enableSignUpWithOpenId || enableLDAP || enableSAML;
     const hasError = Boolean(emailError || nameError || passwordError || serverError || alertBanner);
     const canSubmit = Boolean(email && name && password) && !hasError && !loading;
     const {error: passwordInfo} = isValidPassword('', getPasswordConfig(config), intl);
 
+    const subscribeToSecurityNewsletterFunc = async () => {
+        try {
+            await Client4.subscribeToNewsletter({email, subscribed_content: 'security_newsletter'});
```

Comment:
Nit: This function doesn't need to be `async`, and we don't need to `await` the client call. The catch and console error will still happen without them, and the caller doesn't wait for the result.